src/app/workflow/version-modal/version-modal.component.html

```diff
@@ -40,7 +38,7 @@ <h4 mat-dialog-title>
           </div>
           <div class="form-group form-group-sm">
             <label class="col-sm-3 col-md-3 col-lg-3 control-label">
-              {{ (isService$ | async) ? 'Service Path:' : 'Workflow Path:' }}
+              {{ entryTypeText + ' Path' }}
```

Comment:
I think you lost the colon in changing this. 

I actually think we should get rid of the colons for labels, but that should be across the board. :)